modules/claims_api/spec/mailers/report_monthly_submissions_spec.rb

```diff
@@ -10,12 +10,10 @@
 
       claim = create(:auto_established_claim, :status_established)
       ClaimsApi::ClaimSubmission.create claim:, claim_type: 'PACT', consumer_label: 'Consumer name here'
-      pact_act_data = ClaimsApi::ClaimSubmission.where(created_at: from..to)
 
       described_class.build(
         from,
         to,
-        pact_act_data,
         consumer_claims_totals: [],
         poa_totals: [],
         ews_totals: [],
```

Comment:
@tycol7 Looks like you have a failing test.
![Screenshot 2024-06-21 at 10 22 12 AM](https://github.com/department-of-veterans-affairs/vets-api/assets/25069483/e85f4455-cfbe-46ea-ba5e-e9e1409f4856)

Reply:
Good catch - looks like a flaky test. I [added `match_array`](https://github.com/department-of-veterans-affairs/vets-api/pull/17177/commits/030819dddc578101d82da4b723923afc5885e579) to the expected results, so the tests _should_ pass if the array elements come back in a different order. Is the test passing for you now?